src/PolyhedralGeometry/Cone/properties.jl

```diff
@@ -486,6 +534,29 @@ _generator_matrix(::Val{_hilbert_generator}, C::Polymake.BigObject; homogenized=
 _matrix_for_polymake(::Val{_hilbert_generator}) = _generator_matrix
 
 
+@doc raw"""
+    contains(C0::Cone, C1::Cone)                           
+    
+Check whether `C0` contains `C1`.
+                                             
+# Examples                 
+```jldoctest                                                                                
+julia> C0 = positive_hull([1 0; 0 1])
+Polyhedral cone in ambient dimension 2
+
+julia> C1 = positive_hull([1 1])
+Polyhedral cone in ambient dimension 2
+
+julia> contains(C0, C1)
+true
+
+julia> contains(C1, C0)
+false
+```
+"""
+contains(C0::Cone{T}, C1::Cone{T}) where T<:scalar_types = Polymake.polytope.included_polyhedra(pm_object(C1), pm_object(C0))::Bool
+
+
 @doc raw"""
     contains(C::Cone, v::AbstractVector)
```

Comment:
BTW I think this `contains` really should be `in`

Reply:
I also changed this for the vector containment.